ISLE/main.cpp

```diff
@@ -49,7 +49,7 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
   // Throw error if sound unavailable
   if (!soundReady) {
     MessageBoxA(NULL, "\"LEGO\xAE Island\" is not detecting a DirectSound compatible sound card.  Please quit all other applications and try again.",
-      "Lego Island Error",0);
+      "Lego Island Error", MB_OK | MB_APPLMODAL);
```

Comment:
Is there a purpose to specifying `MB_APPLMODAL` explicitly?

Reply:
No purpose, both `MB_OK` and `MB_APPLMODAL` are defined as `0`.